tests/20-rdfc-verify.js

```diff
@@ -1,84 +1,183 @@
 /*!
- * Copyright 2023 Digital Bazaar, Inc.
+ * Copyright 2024 Digital Bazaar, Inc.
  * SPDX-License-Identifier: BSD-3-Clause
  */
-import {verificationFail, verificationSuccess} from './assertions.js';
-import {endpoints} from 'vc-test-suite-implementations';
+
+import {
+  verificationFail, verificationSuccess
+} from './assertions.js';
+import {
+  endpoints as registeredImplementations
```

Comment:
I liked this name, I think remoteImplementations might also work, but don't change it.